### PR TITLE
Minor changes to holistic app

### PR DESCRIPTION
**Pull request by AndrewBladon**
# Description

In this PR, the names of some API calls have been updated to reflect recent changes to the GeometryEditor API. The interface has also been modified slightly to add a margin to the element containing the buttons (see image below). The change to the interface was intended to prevent some of the labels and buttons (e.g. `Interaction Edit` label) being truncated by the right side of the view.

image

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GeometryEditorOnMap/GeometryEditorOnMap.cpp [hunk 1]
```diff
@@ -277,12 +277,12 @@ void GeometryEditorOnMap::connectSignals()
...
-  connect(m_geometryEditor, &GeometryEditor::selectionChanged, this, [this]()
+  connect(m_geometryEditor, &GeometryEditor::selectedElementChanged, this, [this]()
           {
             emit canDeleteSelectionChanged();
           });
 
-  connect(m_geometryEditor, &GeometryEditor::startedChanged, this, [this]()
+  connect(m_geometryEditor, &GeometryEditor::isStartedChanged, this, [this]()
           {
             emit geometryEditorStartedChanged(); // TODO this event isn't hooked up https://devtopia.esri.com/runtime/bistromath/issues/1704
```
Comment (gayl0000) [inline ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GeometryEditorOnMap/GeometryEditorOnMap.cpp:287]: need to remove comment

Comment (gayl0000) [reply ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GeometryEditorOnMap/GeometryEditorOnMap.cpp:287]: we should also remove ln [146 ](https://github.com/gayl0000/arcgis-runtime-samples-qt/pull/2/files#diff-72f3e1c2fbc4c2a160d06aea782f12c92e46b96d1aee00d557a54d79ee2c2260L146) and ln147

Comment (AndrewBladon) [reply ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GeometryEditorOnMap/GeometryEditorOnMap.cpp]: Change made in https://github.com/gayl0000/arcgis-runtime-samples-qt/pull/2/commits/5b29b070af379cd57a226ef24213f3c6ffaa469e.

**Comment by AndrewBladon** [comment]
Thanks for the additional comments @gayl0000.

I have also made one additional change, which I would like your thoughts on. Currently, the check boxes for the configuration settings are always active. I have changed this so that the check boxes are only active when a `GeometryEditor` session is started. Looking at the way the checkboxes are implemented (i.e. modify the `m_configuration` property), I think this is ok. It saves people getting confused about why the boxes don't do anything while there isn't an active `GeometryEditor` session.

Please let me know if you are happy with the changes in this PR. I will then merge.

File: ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GeometryEditorOnMap/GeometryEditorOnMap.qml [hunk 2]
```diff
@@ -344,6 +344,7 @@ Item {
                         onCheckedChanged: {
                             model.toggleInteractionEdit(checked);
                         }
+                        enabled: model.geometryEditorStarted
```
Comment (gayl0000) [inline ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GeometryEditorOnMap/GeometryEditorOnMap.qml:347]: disabling is unecessary, was there a reason for it? it is possible (and even desireable) to set the configuration before starting

Comment (AndrewBladon) [reply ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GeometryEditorOnMap/GeometryEditorOnMap.qml:347]: Thanks @gayl0000. When I looked at the implementation of the configuration properties yesterday, I thought that changing the configuration when there wasn't an active `GeometryEditor` would have no impact. However, following your comment I've had another look at the implementation and see that I was incorrect. I'll revert these changes.

**Comment by AndrewBladon** [comment]
@gayl0000, https://github.com/gayl0000/arcgis-runtime-samples-qt/pull/2/commits/e68fa286876788b9af82da044f3e46bdd60789f6 has been reverted in https://github.com/gayl0000/arcgis-runtime-samples-qt/pull/2/commits/6fa62748f943ae43a101f9513c63972053e91a6a. Let me know if you are happy for me to merge this PR.

**Review by gayl0000 — APPROVED** [review]
thanks @AndrewBladon, looks good

**Pull request merged**